exercises/practice/crypto-square/test_crypto_square.c

```diff
@@ -19,9 +19,19 @@ static void test_empty_text_res_in_an_empty_ciphertext(void)
    free(res);
 }
 
-static void test_lowercase(void)
+static void normalization_res_in_empty_plaintext(void)
```

Comment:
Test function names should start with 'test_'

```suggestion
static void test_normalization_results_in_empty_plaintext(void)
```